security/src/main/java/no/difi/meldingsutveksling/IntegrasjonspunktNokkel.java

```diff
@@ -88,22 +89,20 @@ protected KeyStore loadKeyStore() {
         String password = properties.getPassword();
         Resource path = properties.getPath();
 
-        KeyStore.ProtectionParameter protectionParameter = new KeyStore.PasswordProtection(password.toCharArray());
-
         try{
-            KeyStore.Builder builder;
+            KeyStore keyStore = KeyStore.getInstance(type);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "keyStore" which hides the field declared at line 21. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AHiddenFieldCheck)